examples/async-servlet/README_pt.adoc

```diff
@@ -0,0 +1,54 @@
+= Async Servlet
+:index-group: Servlet
+:jbake-type: page
+:jbake-status: published
+
+== Async Servlet
+
+Servlets pode ser executado assincronamente - isso pode ser util se seu servlet executa operações de longa execução, como chamar
```

Comment:
I think that "Servlets podem" is better than "Servlets pode".
"pode ser util", the correct is "útil"